sdk/core/azure-core/src/main/java/com/azure/core/http/policy/RetryPolicy.java

```diff
@@ -17,19 +17,19 @@
 
 /**
  * A pipeline policy that retries when a recoverable HTTP error occurs.
+ * @see RetryPolicyOptions
  */
 public class RetryPolicy implements HttpPipelinePolicy {
 
-    private static final String RETRY_AFTER_MS_HEADER = "retry-after-ms";
```

Comment:
You need to update the AppConfiguration RetryPolicy creation to use the `retry-after-ms` header with `ChronoUnit.MILLIS` to maintain current functionality after this change.